R/tribble.R

```diff
@@ -27,19 +27,75 @@
 #'   "b", 4:6
 #' )
 tribble <- function(...) {
+  data <- extract_frame_data_from_dots(...)
 
+  if (length(data$frame_rest) == 0) {
```

Comment:
But you could do here:

```r
if (length(data$frame_rest) == 0) {
  data$frame_rest <- logical()
}
```

I prefer to avoid shortcuts for edge cases if at all possible, because they are easy to overlook if the code needs to be edited.